--- openssh.spec.old	2017-11-04 21:49:51.931587953 +0000
+++ openssh.spec	2017-11-04 21:50:57.637910107 +0000
@@ -279,6 +279,9 @@
 # Amazon changes to default config
 Patch10000: openssh-6.6p1-amzn.patch
 
+# Disable PRIV_END checks
+Patch100000: openssh-6.6p1-privend.patch
+
 License: BSD
 Group: Applications/Internet
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
@@ -560,6 +563,8 @@
 
 %patch10000 -p1 -b .amzn
 
+%patch100000 -p1 -b .privend
+
 autoreconf
 pushd pam_ssh_agent_auth-%{pam_ssh_agent_ver}
 autoreconf
